src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/ContainerRecord.java

```diff
@@ -20,14 +21,16 @@
     
     private final String host;
     private final String containerId;
+    private final String imageId;
```

Comment:
`DockerRunPtrFingerprintFacet` already has this. `DockerRunFingerprintFacet` should have a `readResolve` that sets it back on the records.